crates/apub/src/objects/post.rs

```diff
@@ -70,16 +73,13 @@ impl ToApub for Post {
       set_content_and_source(&mut page, &body)?;
     }
 
-    // TODO: hacky code because we get self.url == Some("")
```

Comment:
hmm.. if there are blank-string URL values in the database, I think this would mean db deserialization would fail before we get here. Is that an acceptable behavior until #602 is addressed? Or maybe I should just throw in a migration to clean up those bad values in this PR too?

Reply:
ok, I added a migration to do that